terraform/groups/ecs-service/main.tf

```diff
@@ -61,15 +61,16 @@ module "ecs-service" {
   healthcheck_matcher            = local.healthcheck_matcher
 
   # Docker container details
-  docker_registry   = var.docker_registry
-  docker_repo       = local.docker_repo
-  container_version = var.confirmation_statement_api_version
-  container_port    = local.container_port
+  docker_registry           = var.docker_registry
+  docker_repo               = local.docker_repo
+  container_version         = var.confirmation_statement_api_version
+  container_port            = local.container_port
 
   # Service configuration
   service_name                       = local.service_name
   name_prefix                        = local.name_prefix
   desired_task_count                 = var.desired_task_count
+  min_task_count                     = var.min_task_count
```

Comment:
On line 84 of this file, I'd remove read_only_root_filesystem = false, as the latest module update should address this.